/*const promise = new Promise((resolve, reject) =>{
    setTimeout(() =>{
        resolve('ok');
    },1000);
});

promise
.then((response) =>{
    console.log(response);
})
.catch((error) =>{
    console.error(error);
});
*/
